test/test_tensor.py

```diff
@@ -55,14 +57,26 @@ def test_gradcheck(self):
     W = np.random.RandomState(1337).random((10, 5))
     x = np.random.RandomState(7331).random((1, 10)) - 0.5
 
-    tiny_x = Tensor(x)
-    tiny_W = Tensor(W)
+    tiny_x = Tensor(x, gpu=self.gpu)
+    tiny_W = Tensor(W, gpu=self.gpu)
     tiny_func = lambda x: x.dot(tiny_W).relu().logsoftmax()
 
     self.assertTrue(gradcheck(tiny_func, tiny_x))
 
     # coarse approx. since a "big" eps and the non-linearities of the model
     self.assertFalse(gradcheck(tiny_func, tiny_x, eps = 0.1))
 
+
+@unittest.skipUnless(GPU, "Requires GPU")
+class TestTinygradGPU(TestTinygrad):
+  gpu = True
+
+  @unittest.skip("float32 not supported on GPU")
```

Comment:
Hmm, float32 is actually the only thing supported on GPU. Is this comment wrong?

Reply:
Yep, float64*. Resolved.